frontend/lib/pages/food_item_page.dart

```diff
@@ -19,25 +50,205 @@ class _FoodItemPageState extends State<FoodItemPage> {
     );
   }
 
+  Future<void> createFood() async {
```

Comment:
I think we should have this in a separate class since I want to abstract the backend logic. But, this can be done in a future pr if you prefer.